content/Basic Cryptography/DES.md

```diff
@@ -0,0 +1,191 @@
+## Intro
+DES is a symmetric key block cipher that uses the same key for both encryption and decryption, processing data in fixed-size blocks.
+> [!note]
+> DES encrypts 64-bit plaintext using a 56-bit key to produce a 64-bit ciphertext.
+
+## History
+In 1973, the National Bureau of Standards (NBS) in the United States recognized the need for a standardized encryption method and called for proposals for what would become the DES (Data Encryption Standard). In 1977, NBS selected a modified version of an encryption algorithm called Lucifer, developed by IBM, to become DES.
+The National Security Agency (NSA) made modifications to Lucifer, including reducing the key length from 128 bits to 56 bits and altering the structure of the S-Box. This led to speculation that the U.S. intelligence agencies could more easily decrypt messages encrypted with DES. Despite these suspicions, DES went on to become the most widely used block cipher worldwide for 30 years after its publication.
+When DES was first introduced in the 1970s, a 56-bit key length provided adequate security. However, with advancements in computing, DES is no longer considered secure. To address these limitations, multiple DES techniques were introduced, notably Triple DES (3DES), which applies DES three times and provides 112 bits of security, making it more robust.
```

Comment:
EFF's cracking hardware and distributed.net were also important events contributed to its deprecation.